src/content/docs/style-guide/images/embed-videos.mdx

```diff
@@ -54,6 +54,13 @@ To upload videos to Wistia:
   />
   ```
 
+<Callout variant="tip">
+  For "What's new" posts that use `.md` files, you can't use the video component. Instead, you'd use an `iframe` tag like this: 
+  ```html
+  <iframe width="560" height="315" src="https://fast.wistia.net/embed/iframe/o8r4tfsa8l" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Should we replace `o8r4tfsa8l` with a placeholder value?

Reply:
I replaced the code with a placeholder.